src/main/java/net/backupcup/mcde/util/AdvancementList.java

```diff
@@ -0,0 +1,43 @@
+package net.backupcup.mcde.util;
+
+import java.util.List;
+import java.util.Map;
+
+import blue.endless.jankson.JsonArray;
+import blue.endless.jankson.JsonObject;
+import blue.endless.jankson.annotation.Deserializer;
+import blue.endless.jankson.annotation.Serializer;
+import blue.endless.jankson.api.SyntaxError;
+import net.minecraft.advancement.Advancement;
+import net.minecraft.util.Identifier;
+
+public class AdvancementList extends IdentifierGlobList<Advancement> {
+    public AdvancementList(String... globs) {
+        super(globs);
+    }
+
+    protected AdvancementList(Map<String, List<Glob>> namespaces) {
+        super(namespaces);
+    }
+
+    @Override
+    public Identifier getId(Advancement obj) {
```

Comment:
Again: Description is key. Consider `Advancement advancement` rather than `Advancement obj`.